src/test/java/org/carlspring/cloud/storage/s3fs/util/CopyDirVisitor.java

```diff
@@ -69,12 +69,12 @@ private Path appendPath(Path to)
     {
         Path targetPath = toPath;
 
-        // sacamos el path relativo y lo recorremos para
-        // añadirlo al nuevo
+        // Take out the relative path and traverse through it
+        // and add new to it..
```

Comment:
```suggestion
        // We take the relative path and travers through it
        // to add it again
```

Reply:
@ptirator: What sounds better here?

Reply:
@carlspring: Sacamos el path relativo y lo recorremos para añadirlo al nuevo = Get the relative path and traverse it to add it to the new path.

Reply:
:+1: Thanks, @ptirador!

@viveksahu26: Could you please apply this change?